packages/twenty-server/src/modules/messaging/message-import-manager/drivers/microsoft/services/microsoft-get-messages.service.ts

```diff
@@ -80,7 +80,7 @@ export class MicrosoftGetMessagesService {
 
     const messages = parsedResponses.map((response) => {
       if ('error' in response) {
-        this.microsoftHandleErrorService.handleMicrosoftMessageFetchError(
+        this.microsoftHandleErrorService.handleMicrosoftBatchResponseError(
```

Comment:
Remark: Unless I'm mistaken this throws at first fail
Wondering if that's that expected regarding the batch naming, maybe we should silently aggreg all errors and throw an exceptions afterwards ? ( too at least have logs on back ) ?

Reply:
That could be an improvement indeed ! Results will be the same though because we put back in the queue these messageIds to be re-fetched